f3.go

```diff
@@ -62,16 +63,19 @@ type F3 struct {
 // New creates and setups f3 with libp2p
 // The context is used for initialization not runtime.
 func New(_ctx context.Context, manifest manifest.Manifest, ds datastore.Datastore, h host.Host,
-	ps *pubsub.PubSub, verif gpbft.Verifier, ec ec.Backend, diskPath string) (*F3, error) {
+	ps *pubsub.PubSub, verif gpbft.Verifier, ecBackend ec.Backend, diskPath string) (*F3, error) {
 	runningCtx, cancel := context.WithCancel(context.WithoutCancel(_ctx))
+
+	// concurrency is limited to half of the number of CPUs, and cache size is set to 256 which should be enough for most use cases
```

Comment:
Why 256? Because we don't expect that much diversity in tipset keys within a hand full of instances? 

I can imagine a future reader to question what "most use cases" means.